examples/telnet/as400_power_actions.js

```diff
@@ -51,52 +38,11 @@ function getAS400Info() {
             console.error("error while executing command: " + telnetParams.command);
             failure(err);
         }
-        d.resolve(out.replace(/\\x1B\\[[0-9;]*[a-zA-Z]/g, " "));
+        d.resolve(out);
```

Comment:
This and the validate mechanism means that ALL devices that respond to the telnet command without an error are assumed to be capable of the subsequent actions (reboot/shutdown)

I don't think this is a correct assumption?